geoportail.lu.xcodeproj/project.pbxproj

```diff
@@ -551,9 +551,9 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = geoportail.lu/geoportail.lu.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				CODE_SIGN_STYLE = Automatic;
```

Comment:
I don't think this is intentional.

Reply:
I kept unstaged your modifications on the project. Didn't touch that.